src/ProjectOrigin.WalletSystem.Server/CommandHandlers/VerifySliceCommandHandler.cs

```diff
@@ -128,6 +132,13 @@ private async Task InsertIntoWallet(VerifySliceCommand receivedSlice, GranularCe
             await _unitOfWork.CertificateRepository.InsertCertificate(certificate);
         }
 
+        foreach (var hashedAttribute in receivedSlice.HashedAttributes)
+        {
+            if (certificateProjection.Attributes.Any(att => att.Key == hashedAttribute.Key
```

Comment:
If this is false, should it then be logged?

Reply:
I have added a log statement, good catch